durable_storage/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2025 Trilitech <contact@trili.tech>
+//
+// SPDX-License-Identifier: MIT
+
+//! Implementation of the RISC-V PVM durable storage's persistence layer.
+//!
+//! A durable storage is responsible for providing a crash-resistant key-value store.
+//!
+//! The main components of the durable storage are:
+//! - **Repository**: This will correspond to a directory on disk, and will act as the storage place for
```

Comment:
Huge nit but I think using present tense in this comment makes sense, even if the components aren't implemented. Their role won't presumably change with the implementation.

Reply:
This makes a lot of sense actually 👍  [diff](https://github.com/tezos/riscv-pvm/compare/ab75ddebaaf8dba77878acdcd866f9597cae8886..093af426072fb32197307f6d7d62149469aa37d6)

Reply:
The "repository" explanation still uses future tense.